stories/primer/tab_nav_component_stories.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Primer::TabNavComponentStories < ViewComponent::Storybook::Stories
+  layout "storybook_preview"
+
+  story(:tab_nav) do
+    controls do
+      with_panel true
+    end
+
+    content do |c|
+      c.tab(selected: true, title: "Tab 1") { "Panel 1" }
+      c.tab(title: "Tab 2") { "Panel 2" }
+      c.tab(title: "Tab 3") { "Panel 3" }
```

Comment:
Is there a way to add an open text field so users could pass in arbitrary `classes`? It would make testing issues like https://github.com/primer/view_components/issues/280 much easier!

Reply:
unfortunately we can only control the **parent** component in Storybook :(
It's a `view_component_storybook` limitation, but I also don't know if component previews allow controlling a component's slot
@BlakeWilliams thoughts?